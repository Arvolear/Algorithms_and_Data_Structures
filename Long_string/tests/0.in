13414917660376734308595991279
729453