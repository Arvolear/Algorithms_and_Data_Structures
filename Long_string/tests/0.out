13414917660376734308596720732
13414917660376734308595261826
9785551932114789971608271626440387
18390379723404707785965